### ISSUE TITLE
ERROR: CreateProcessEntryCommon - unable to login to WSL after import

### ISSUE THREAD

**Issue by BeErikk** [microsoft/WSL#11343]
### Windows Version

Microsoft Windows [Version 10.0.22631.3296]

### WSL Version

 2.1.5.0

### Are you using WSL 1 or WSL 2?

- [ ] WSL 2
- [X] WSL 1

### Kernel Version

Linux version 4.4.0-22621-Microsoft (Microsoft@Microsoft.com) (gcc version 5.4.0 (GCC) ) #2506-Microsoft Fri Jan 01 08:00:00 PST 2016

### Distro Version

Ubuntu 22.04

### Repro Steps

New computer, new Windows 11 Pro instance, no insider versions.
I have one distribution installed, Ubuntu WSL1, previously created in Windows 10. It was created manually using "the server method" outside the system disk. 

Distro imported by:
`wsl.exe --import distro "<path>\distro" "<path>\distro.tar"`

There is a `/etc/wsl.conf` with 
`[user]`
`default=username`

### Expected Behavior

`wsl ~ -u username`
`Welcome to Ubuntu 22.04.4 LTS (GNU/Linux 4.4.0-22621-Microsoft x86_64)`
`...`

### Actual Behavior

`wsl ~ -u username`
`<3>WSL (95) ERROR: CreateProcessEntryCommon:496: chdir(/home/username) failed 13`
`<3>WSL (95) ERROR: CreateProcessEntryCommon:505: execvpe /bin/bash failed 13`
`<3>WSL (95) ERROR: CreateProcessEntryCommon:508: Create process not expected to return`

**Comment by BeErikk** [comment]
The first error seems to be while trying to change WSL directory to the current directory of execution.

**Comment by BeErikk** [comment]
Logging in as `root` with
`wsl.exe ~ -u root`
works as expected.

**Comment by BeErikk** [comment]
I also tried to create a new user account by as `root` issue
`useradd --create-home --shell /usr/bin/bash --user-group --groups  adm,dialout,cdrom,floppy,sudo,audio,dip,video,plugdev,netdev --password $(read -sp Password: pw ; echo $pw | openssl passwd -1 -stdin) newuser`
then
`wsl ~ -u newuser`

Same result.

**Comment by OneBlue** [comment]
Thank you @BeErikk. Based on the output it looks like your distro might be missing some files. 

What's the output of `wsl -d <distro> -u root file /bin/bash`  and `wsl -d <distro> -u root ls -la /home`  ?

**Comment by BeErikk** [comment]
Thanks for answering @OneBlue!

`wsl.exe -d ubuntu-wsl1 -u root file /bin/bash`
`Welcome to Ubuntu 22.04.4 LTS (GNU/Linux 4.4.0-22621-Microsoft x86_64)`
`...`

`wsl.exe -d ubuntu-wsl1 -u root ls -la /home`
`total 0`
`d--------- 1 root    root    4096 Mar 22 15:20 .`
`drwxr-xr-x 1 root    root    4096 Jan  1  1970 ..`
`drwxr-x--- 1 BeErikk BeErikk 4096 Mar 22 15:20 BeErikk`
`d--------- 1 root    root    4096 Mar  2 12:09 <realuser>`

`BeErikk` is the extra user account I tried to add, <realuser> is my Windows account name. Computer and I as a user are also part of Active Directory.

**Comment by OneBlue** [comment]
Ok so it looks like 'username', which is what wsl.conf points to doesn't exist. You need to update /etc/wsl.conf to map to a user that actually exists and that has a home directory (like BeErikk or <realuser>).

**Comment by OneBlue** [comment]
What was the output of: 

```

useradd --create-home --shell /usr/bin/bash --user-group --groups  adm,dialout,cdrom,floppy,sudo,audio,dip,video,plugdev,netdev --password $(read -sp Password: pw ; echo $pw | openssl passwd -1 -stdin) newuser
``` ?  This command might have just failed and ended up not actually creating the account

**Comment by BeErikk** [comment]
Thanks very much @OneBlue!

I think even if I now have a new machine and upgraded to Windows 11, my Windows username is exactly the same since it is stored in Active Directory (if it matters) and that name is also the name I used in WSL. Everything in WSL should be as I left it on my old Windows 10 machine. The migration should work as I understand it. It didn't and I resorted to installing a new WSL 1 Ubuntu instance.

Again thank you for your effort!
